--- gob-1.0.12/src/Makefile.am.orig	Sun Oct 21 10:36:43 2001
+++ gob-1.0.12/src/Makefile.am	Mon Feb  4 05:25:26 2002
@@ -1,15 +1,11 @@
 #YACCFLAGS += -d -t
-YFLAGS += -d -t
+YFLAGS = -d -t
 SUBDIRS =
 
-CFLAGS += -Wall -Wpointer-arith -Wmissing-prototypes -Wmissing-declarations
+INCLUDES = $(GLIB_CFLAGS) -I$(includedir)
 
-CFLAGS += -DPKGDATADIR=\"$(pkgdatadir)\"
-
-INCLUDES = @GLIB_CFLAGS@ -I$(includedir)
-
-bin_PROGRAMS = @DOINSTGOB@
-noinst_PROGRAMS = @NOINSTGOB@
+bin_PROGRAMS = $(DOINSTGOB)
+noinst_PROGRAMS = $(NOINSTGOB)
 EXTRA_PROGRAMS = gob
 
 BUILT_SOURCES = parse.h parse.c lexer.c
@@ -31,12 +27,12 @@
 gob_LDADD = \
 	-lm \
 	$(GLIB_LIBS) \
-	@LEXLIB@
+	$(LEXLIB)
 
 # we might want popt back in the future
-#	@POPT_LIB@
+#	$(POPT_LIB)
 
-CLEANFILES += $(BUILT_SOURCES) gtk-weird-button.* gtk-weird-button-private.h
+CLEANFILES = $(BUILT_SOURCES) gtk-weird-button.* gtk-weird-button-private.h
 
 EXTRA_DIST = $(BUILT_SOURCES) test.gob generate_treefuncs.pl treefuncs.def
 
